modules/userdirectory-canvas/src/main/resources/OSGI-INF/canvas-user-factory.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We tried using a common format for all these files.
Not that it's checked anywhere ;-)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
               name="org.opencastproject.userdirectory.canvas.CanvasUserProviderFactory"
               immediate="true">
  <implementation class="org.opencastproject.userdirectory.canvas.CanvasUserProviderFactory" />
  <property name="service.pid"
            value="org.opencastproject.userdirectory.canvas" />
  <property name="service.description"
            value="Provides Canvas user directory instances" />
  <service>
    <provide interface="org.osgi.service.cm.ManagedServiceFactory"/>
  </service>
  <reference name="orgDirectory"
             interface="org.opencastproject.security.api.OrganizationDirectoryService"
             bind="setOrgDirectory" />
</scr:component>
```